python/lsst/obs/base/gen2to3/repoConverter.py

```diff
@@ -205,7 +207,7 @@ class RepoConverter(ABC):
     def __init__(self, *, task: ConvertRepoTask, root: str, run: Optional[str],
                  subset: Optional[ConversionSubset] = None):
         self.task = task
-        self.root = root
+        self.root = os.path.realpath(root)
```

Comment:
I had to look up the difference between `realpath` and `abspath` again, and came across this: https://stackoverflow.com/questions/37863476/why-would-one-use-both-os-path-abspath-and-os-path-realpath which pointed out that neither will expand `~` unless you do (e.g.) `os.path.realpath(os.path.expanduser(root))`.  Is this an edge case that we should care about?

Reply:
Well if people are going to do `~/mygen2root` then yes I need expanduser in there... 😉